RunLog/RunLog/Resources/DesignSystem.swift

```diff
@@ -0,0 +1,204 @@
+//
+//  DesignSystem.swift
+//  RunLog
+//
+//  Created by 심근웅 on 3/14/25.
+//
+
+import Foundation
+import UIKit
+
+// MARK: - 컬러
+enum DesignSystemColor {
```

Comment:
```suggestion
enum RLColor {
```
코드를 작성할때 DesignSystem. ... 이렇게 작성해야할텐데 너무 길어보여서 제안드립니다. 저희 앱 내에서 사용하는 디자인 시스템이니 RunLog -> RLColor 요런식으로 하는건 어떤지 제안드립니다.